web-portal/components/AdminEventsList.vue

```diff
@@ -18,7 +18,14 @@
           <small v-if="calendar_event.date_times.length > 2">(and {{ calendar_event.date_times.length - 2 }} more)</small>
           <template v-if="calendar_event.category && calendar_event.category==='online-resource'">Online Resource</template>
         </td>
-        <td><v-btn nuxt :to="{ name: 'admin-event-edit-id', params: { id: calendar_event.id } }">Edit</v-btn></td>
+        <td>
+          <ii-form-button
+            style-type="light"
+            @click="onEditClicked(calendar_event.id)"
+          >
+            Edit
+          </ii-form-button>
+        </td>
```

Comment:
I think the Cypress tests are failing because this `v-btn` was actually rendering as a link (because of the `nuxt` and `:to` props).

Reply:
yeah makes sense, will have a look

Reply:
I debated trying to implement a link version of this but didn't really seem worth it

Reply:
pushed up a change that should fix this. I went ahead and added a test-id to preserve the spirit of one test which queries for a specific event. I think there was only one even on the page, so I could have simplified it to just check for button, but I like the specificity in that case

Reply:
At some point (but it can wait until we rework this for a "design system") I want to revisit this and support rendering as an `a`.